sds/Cargo.toml

```diff
@@ -22,9 +22,10 @@ ahash = "0.8.7"
 # Overriding with fix repo due to: https://github.com/seiflotfy/rust-farmhash/pull/16
 farmhash = { git = "https://github.com/fuchsnj/rust-farmhash", rev = "82d80b689d65fbd378b13deff10cdd07794df64e" }
 nom = "7.1.3"
-regex = "1.9.5"
-regex-automata = "0.4.4"
-regex-syntax = "0.7.5"
+regex = { git = "https://github.com/fbryden/regex", tag = "1.9.5"}
```

Comment:
We should probably continue using the un-forked `regex` crates for normal regex things (so we continue receiving updates). Only the `pool` needs the forked. You can pull in _just_ `regex-automata` from the forked repo and rename it to something unique to it doesn't conflict with the "real" `regex-automata`. We should also include a link to the issue in the `regex` repo to make this configurable which would allow us to stop using the fork.